apis/validator/README.md

```diff
@@ -1,6 +1,6 @@
 # Ethereum 2.0 Phase 0 -- Beacon Node API for Validator
 
-**Notice**: This document is a work-in-progress for researchers and implementers. This is an accompanying document to [Ethereum 2.0 Phase 0 -- Honest Validator](https://github.com/ethereum/eth2.0-specs/blob/master/specs/validator/0_beacon-chain-validator.md) that describes an API exposed by the beacon node, which enables the validator client to participate in the Ethereum 2.0 protocol.
+**Notice**: This document is a work-in-progress for researchers and implementers. This is an accompanying document to [Ethereum 2.0 Phase 0 -- Honest Validator](https://github.com/ethereum/eth2.0-specs/blob/master/specs/phase0/validator.md) that describes an API exposed by the beacon node, which enables the validator client to participate in the Ethereum 2.0 protocol.
```

Comment:
```suggestion
**Notice**: This document is a work-in-progress for researchers and implementers. This is an accompanying document to [Ethereum 2.0 Phase 0 -- Honest Validator](https://github.com/ethereum/eth2.0-specs/blob/v0.11.3/specs/phase0/validator.md) that describes an API exposed by the beacon node, which enables the validator client to participate in the Ethereum 2.0 protocol.
```
I'll update all versions once we merge current open PRs

Reply:
this shouldn't be on (https://github.com/ethereum/eth2.0-specs/blob/v0.12.1/specs/phase0/validator.md)?

Reply:
Everything is currently on 0.11.3  butt yeah it should be on 0.12 but I'm waiting to integrate open prs so i can change it all in one batch